### DISCUSSION TITLE
Getting runtime errors in docker slim container

### DISCUSSION THREAD

**Discussion by shubh1608** [slimtoolkit/slim#387] [Q&A]
I ran below command, It reduced my docker image size from 3.09 GB to 1.78 GB, Also note that I included the python packages directory as on the initial attempt docker-slim removed the 'gdown' package and it triggered the error at runtime in the container. 

> docker-slim build --include-path /usr/local/lib/python3.8/site-packages IMAGE-NAME

Now after running slim container, I am getting the below error which seems to be related to Pillow package, looks like docker-slim has affected the Pillow package. I have verified that this error is not coming with the original image i.e. a non slim one.

> Traceback (most recent call last): File "/usr/local/lib64/python3.8/site-packages/flask/app.py", line 2525, in wsgi_app response = self.full_dispatch_request() File "/usr/local/lib64/python3.8/site-packages/flask/app.py", line 1822, in full_dispatch_request rv = self.handle_user_exception(e) File "/usr/local/lib64/python3.8/site-packages/flask/app.py", line 1820, in full_dispatch_request rv = self.dispatch_request() File "/usr/local/lib64/python3.8/site-packages/flask/app.py", line 1796, in dispatch_request return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args) File "/app/api/app.py", line 39, in verifyface img1_np = np.asarray(Image.open(img1.stream).convert('RGB')) File "/usr/local/lib64/python3.8/site-packages/PIL/Image.py", line 3147, in open raise UnidentifiedImageError( PIL.UnidentifiedImageError: cannot identify image file <tempfile.SpooledTemporaryFile object at 0x7fa893b4e220>

Looking for help here. Please suggest how to solve this error.

**Comment by kcq** [comment]
This might be related to the temporary directory management logic in the Pillow package. Try adding an `--include-path` for the temporary directory configured for Pillow in your app.
